import-export-cli/cmd/exportAPI.go

```diff
@@ -58,6 +61,9 @@ var ExportAPICmd = &cobra.Command{
 	Example: exportAPICmdExamples,
 	Run: func(cmd *cobra.Command, args []string) {
 		utils.Logln(utils.LogPrefixInfo + ExportAPICmdLiteral + " called")
+		if exportRevisionNum == "" {
+			fmt.Println("Revision number not provided. Only the working copy without deployment environments will be exported")
```

Comment:
Shall we change this as follows?
```suggestion
			fmt.Println("The revision number is not provided. Only the working copy without deployment environments will be exported.")
```